curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-bracket-notation.md

```diff
@@ -77,15 +77,12 @@ assert(code.match(/testObj\s*?\[('|")[^'"]+\1\]/g).length > 1);
 ## --seed-contents--
 
 ```js
-// Setup
 var testObj = {
   "an entree": "hamburger",
   "my side": "veggies",
   "the drink": "water"
 };
 
-// Only change code below this line
-
 var entreeValue = testObj;   // Change this line
 var drinkValue = testObj;    // Change this line
```

Comment:
Rather than leaving two `Change this line` comments, would it be better to leave the `Only change code below this line` instead? Or remove all of them?

@moT01 thoughts?

Reply:
That is what @ShaunSHamilton had suggested. He probably has a better idea of what is going to trip people up. I think the instructions are clear enough that no comments are needed, but I don't mind it either way.

Reply:
More than anything, it just seems strange to me to have two `Change this line` comments right next to each other.

Reply:
Yea, I didn't look close enough at this one. Those two should definitely go, and we should either leave it at that (with no comments) or add the suggested one back in.

Reply:
I'm also leaning a bit more towards what @ShaunSHamilton suggested -- remove the two `Change this line` comments and leave the `Only change code below this line` comment as is.

We could remove all the comments, but it seems helpful to leave `Only change code below this line` because `testObj` is editable and our tests rely on it.